src/sagemaker/model.py

```diff
@@ -69,6 +69,17 @@
 
 NEO_IOC_TARGET_DEVICES = ["ml_c4", "ml_c5", "ml_m4", "ml_m5", "ml_p2", "ml_p3", "ml_g4dn"]
 
+NEO_MULTIVERSION_UNSUPPORTED = [
```

Comment:
Curious which edge targets do we want to support?

Reply:
https://docs.aws.amazon.com/sagemaker/latest/dg/neo-supported-devices-edge-devices.html